src/templates/index.html

```diff
@@ -96,7 +96,7 @@ <h3>{{ _('On Mac OSX') }}</h3>
 </div>
 <p>
     {% trans trimmed
-        discord='<a target="_blank" href="https://discordapp.com">Discord</a>',
+        discord=external("https://www.facebook.com/groups/1543249585925041", _('Discord')),
```

Comment:
I think you lost the right link in the process :-p

Reply:
does `external` use the `target` part ? if not, maybe just adding `|safe`

Reply:
Yeah `external` always adds `target="_blank"`

Reply:
Okay, so I'll put the right link next commit then

Reply:
Fixed